content/docs/tutorials/ocm-and-gitops/deploying-applications-with-ocm-and-gitops.md

```diff
@@ -103,12 +103,19 @@ flux reconcile source git flux-system
 flux get kustomizations
 ```
 
-### Deploy the OCM Controller
+### Install the OCM Controller
 
-We can use the OCM CLI to install the controller:
+There are two ways to install the `ocm-controller`. One, is by using `ocm controller install` command from the [OCM](https://github.com/open-component-model/ocm) project.
```

Comment:
Do you think we should push for this installation method? Thinking that we will need to keep that also in the new cli then